lib/widgtes/shared/bottom_menu.dart

```diff
@@ -3,74 +3,47 @@ import 'package:shared_preferences/shared_preferences.dart';
 import '../../data/http_helper.dart';
 import '../../data/user.dart';
+import 'package:learning_analytics/views_d/mein_profil.dart';
+import 'package:learning_analytics/views_d/meine_kurse.dart';
+
+import 'package:learning_analytics/views_s/mein_lernen.dart';
+import 'package:learning_analytics/views_s/mein_profil.dart';
+import 'package:learning_analytics/views_s/mein_fortschritt.dart';
 
 @immutable
 class BottomMenu extends StatefulWidget {
-  const BottomMenu({Key? key, required this.index}) : super(key: key);
+  const BottomMenu(
+      {Key? key, this.index = 1, required this.user, this.onSelectTab})
+      : super(key: key);
   final int index;
+  final User user;
+  final onSelectTab;
 
   @override
   State<BottomMenu> createState() => _BottomMenuState();
 }
 
 class _BottomMenuState extends State<BottomMenu> {
   Widget build(context) {
-    return FutureBuilder<String>(
-        future: fetchData(),
-        builder: (context, AsyncSnapshot<String> snapshot) {
-          if (snapshot.hasData) {
-            return CurvedNavigationBar(
-                height: 50,
-                index: widget.index,
-                letIndexChange: (index) => true,
-                backgroundColor: Colors.transparent,
-                color: Colors.transparent,
-                buttonBackgroundColor: Theme.of(context).primaryColorLight,
-                items: [
-                  Icon(Icons.account_circle, color: Colors.black),
-                  Icon(Icons.menu_book_sharp, color: Colors.black),
-                  Icon(
-                    snapshot.data == "Student"
-                        ? Icons.bar_chart
-                        : Icons.add_circle_outline_outlined,
-                    color: Colors.black,
-                  )
-                ],
-                onTap: (int index) {
-                  String view = '';
-                  String role = snapshot.data == "Student" ? "S" : "D";
-                  switch (index) {
-                    case 0:
-                      view = '/meinProfil$role';
-                      break;
-                    case 1:
-                      view = '/meinLernenS';
-                      break;
-                    case 2:
-                      view = role == "S" ? '/meinFortschrittS' : '/meineKurseD';
-                      break;
-                  }
-                  String? currentView = ModalRoute.of(context)?.settings.name;
-                  if (currentView != view) {
-                    Navigator.popAndPushNamed(context, view);
-                  }
-                });
-          }
-          return Text("");
+    return CurvedNavigationBar(
+        height: 50,
+        index: widget.index,
+        letIndexChange: (index) => true,
+        backgroundColor: Colors.transparent,
+        color: Colors.transparent,
+        buttonBackgroundColor: Theme.of(context).primaryColorLight,
+        items: [
+          Icon(Icons.account_circle, color: Colors.black),
```

Comment:
Ist es möglich, dass die Icons in weiß sind, wenn die ausgewählt sind und der Hintergrund somit rot ist.